src/core/include/openvino/op/is_finite.hpp

```diff
@@ -25,6 +25,11 @@ class OPENVINO_API IsFinite : public Op {
     bool visit_attributes(AttributeVisitor& visitor) override;
 
     void validate_and_infer_types() override;
+
+    OPENVINO_SUPPRESS_DEPRECATED_START
+    bool evaluate(const HostTensorVector& outputs, const HostTensorVector& inputs) const override;
```

Comment:
Please don't use deprecated API. Please use new API for such methods.

Reply:
Will do, in progress, found some issues with tests after switch to new API